backend/events/endpoints.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import include, url
```

Comment:
Ok, there's clearly some confusion right now about how we're doing URLs. I think we need to change this, but I'm going to need to look into it a little more.